test-data/unit/stubgen.test

```diff
@@ -168,16 +203,22 @@ x, y = 1, 2
 [out]
 from typing import Any
 
-x = ...  # type: Any
-y = ...  # type: Any
+x: Any
+y: Any
+
+[case testMultipleAssignmentAnnotated]
+x, y = 1, "2" # type: int, str
```

Comment:
> I'm a bit converned that validation is a bit out of scope from the stubgen tool.

Just not crashing on invalid input is already OK, I think. If you can produce a reasonable warning, then it's perfect.